Seminars/Sem.03/Pract.03/Task02.cpp

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <fstream>
+
+using std::cout;
+using std::cin;
+using std::endl;
+using std::ios;
+using std::ofstream;
+using std::ifstream;
+
+const int MAX_COMPANY_NAME_LENGTH = 25;
+const int MAX_INPUT_LENGTH = 256;
+
+struct jobOffer {
+	char companyName[MAX_COMPANY_NAME_LENGTH]; //25
+	int teamCount; //4
+	int paidVacancyDays; //4
+	long long salary; //8
+};
+
+void filterOffers(ifstream& file, long long minSalary);
+void perfectOffer(ifstream& file, int maxCoworkers,
+	int minVacancyDays, int minSalary);
+void readOffersFromConsoleAndSaveInFile(const char* filePath,
+	int requestsCount);
+
+
+int main()
+{
+	readOffersFromConsoleAndSaveInFile("file.dat", 2);
+	ifstream file("file.dat", ios::binary | ios::in);	
+	if (!file.is_open())
+	{
+		cout << "Error opening " << "file.dat" << "." << endl;
+		return -1;

```

Comment:
Avoid return -1?
Waiting on an answer why we shouldn't return -1 in main function